plans/init.pp

```diff
@@ -0,0 +1,97 @@
+# Reboots targets and waits for them to be available again.
+#
+# @param nodes Targets to reboot.
+# @param message Message to log with the reboot (for platforms that support it).
+# @param reboot_delay How long (in seconds) to wait before rebooting. Defaults to 1.
+# @param disconnect_wait How long (in seconds) to wait before checking whether the server has rebooted. Defaults to 1.
+# @param reconnect_timeout How long (in seconds) to attempt to reconnect before giving up. Defaults to 120.
+# @param retry_interval How long (in seconds) to wait between retries. Defaults to 1.
+plan reboot (
+  TargetSpec $nodes,
+  Optional[String] $message = undef,
+  Integer[1] $reboot_delay = 1,
+  Integer[0] $disconnect_wait = 10,
```

Comment:
It looks like the actual default is 10 when it should be 1?

Reply:
Documentation was wrong, I updated it to 10.